inst/unitTests/runit.loadWorkbook.R

```diff
@@ -87,4 +87,9 @@ test.loadWorkbook <- function() {
 	# correct password is specified
 	# Excel2019
 	wb <- loadWorkbook(pwdProtectedFile, password = "mirai")
+
+	# Check that loadWorkbook can handle relative paths (#212)
+	setwd(rsrc("resources"))
```

Comment:
key to the bug is that the working directory is set in R during runtime, and the JVM doesn't completely "follow" that.